std_msgs/Header header

## fleet info for robust payload control
float32  r_jx         ## [m]
float32  r_jy         ## [m]
float32  v_jx         ## [m/s]
float32  v_jy         ## [m/s]
float32  f_Ljx        ## [N]
float32  f_Ljy        ## [N]
float32  f_Ljz        ## [N]
float32  delta_jx     ## [N]
float32  delta_jy     ## [N]
float32  delta_jz     ## [N]
float32  rd_jx        ## [m]
float32  rd_jy        ## [m]
bool emergency        ##